suite-native/graph/src/hooks.ts

```diff
@@ -8,14 +8,52 @@ import {
     useGraphForAccounts,
 } from '@suite-common/graph';
 import { selectAccountByKey, selectAccounts } from '@suite-common/wallet-core';
-import { AccountKey } from '@suite-common/wallet-types';
+import { Account, AccountKey } from '@suite-common/wallet-types';
+import { analytics, EventType } from '@suite-native/analytics';
+
+import { timeSwitchItems } from './components/TimeSwitch';
+
+type HoursValue = number | null;
+
+const useWatchTimeframeChangeForAnalytics = (
+    hoursToHistory: HoursValue,
+    account: Account | null = null,
```

Comment:
`accountSymbol` is all you need. No need to send the whole object

Reply:
fixed in 7c3f39874